qiskit_experiments/framework/backend_data.py

```diff
@@ -0,0 +1,210 @@
+
+    def __init__(self, backend):
+        """Inits the backend and verifies version"""
+        self._backend = backend
+        self._v1 = isinstance(backend, BackendV1)
+        self._v2 = isinstance(backend, BackendV2)
+        if self._v2:
+            self._parse_additional_data()
+
+    def _parse_additional_data(self):
+        # data specific parsing not done yet in qiskit-terra
+        if self._backend._conf_dict["open_pulse"]:
+            if "u_channel_lo" not in self._backend._conf_dict:
+                self._backend._conf_dict["u_channel_lo"] = []  # to avoid terra bug
+            self._pulse_conf = PulseBackendConfiguration.from_dict(self._backend._conf_dict)
+
+    @property
+    def name(self):
+        """Returns the backend name"""
+        if self._v1:
+            return self._backend.name()
+        elif self._v2:
+            return self._backend.name
+        return str(self._backend)
+
+    def control_channel(self, qubits):
+        """Returns the backend control channels"""
+        if self._v1:
+            return self._backend.configuration().control(qubits)
+        elif self._v2:
+            try:
+                return self._backend.control_channel(qubits)
+            except (AttributeError, NotImplementedError):
+                return self._pulse_conf.control_channels[qubits]
+        return None
+
+    @property
+    def granularity(self):
+        """Returns the backend's time constraint granularity"""
+        try:
+            if self._v1:
+                return self._backend.configuration().timing_constraints.get("granularity", 1)
+            elif self._v2:
+                return self._backend.target.granularity
+        except AttributeError:
+            return 1
+        return 1
+
+    @property
+    def min_length(self):
+        """Returns the backend's time constraint minimum duration"""
+        try:
+            if self._v1:
+                return self._backend.configuration().timing_constraints.get("min_length", 0)
+            elif self._v2:
+                return self._backend.target.min_length
+        except AttributeError:
+            return 0
+        return 0
+
+    @property
+    def pulse_alignment(self):
+        """Returns the backend's time constraint pulse alignment"""
+        try:
+            if self._v1:
+                return self._backend.configuration().timing_constraints.get("pulse_alignment", 1)
+            elif self._v2:
+                return self._backend.target.pulse_alignment
+        except AttributeError:
+            return 1
+        return 1
+
+    @property
+    def aquire_alignment(self):
+        """Returns the backend's time constraint acquire alignment"""
+        try:
+            if self._v1:
+                return self._backend.configuration().timing_constraints.get("aquire_alignment", 1)
+            elif self._v2:
+                return self._backend.target.aquire_alignment
+        except AttributeError:
```

Comment:
```suggestion
    def acquire_alignment(self):
        """Returns the backend's time constraint acquire alignment"""
        try:
            if self._v1:
                return self._backend.configuration().timing_constraints.get("acquire_alignment", 1)
            elif self._v2:
                return self._backend.target.acquire_alignment
        except AttributeError:
```

Reply:
That's a tricky one... In the current implementation of `Target` in Terra, the typo is used:

https://github.com/Qiskit/qiskit-terra/blob/244fd840a85ba2c8c13d29bd72c20c51c6722481/qiskit/transpiler/target.py#L249

But I'm not sure about the `BackendV1` configuration, I'll double-check.

Reply:
:facepalm:

Reply:
I opened https://github.com/Qiskit/qiskit-terra/pull/8423 for this but maybe we need to work with the misspelling in the mean time.

Reply:
I'll add an attempt to get the correct field in addition to the misspelled field.